tutor/specs/e2e/my-courses.e2e.ts

```diff
@@ -117,6 +118,46 @@ describe('My Courses', () => {
             await expect(page).toHaveSelector('testEl=existing-teacher-screen')
             await expect(page).not.toHaveText('testEl=course-card', 'ConceptCoach', { timeout: 100 })
         })
+
+        it.only('deletes offering', async () => {
+            let courseIdWithStudents;
+            const offeringIdWithStudents = mock.current.bootstrapData.offerings[0].id
+            const offeringIdWithEmptyStudents = mock.current.bootstrapData.offerings[1].id
+            mock.current.bootstrapData.courses.forEach((c: Course, index: number) => {
+                if(index === 0) {
+                    // leave one course with students
+                    courseIdWithStudents = c.id;
+                    c.offering_id = offeringIdWithStudents
+                } else {
+                    // rest of the course give a current offering, and set number of students to 0
+                    c.offering_id = offeringIdWithEmptyStudents
+                    c.periods.forEach(p => p.num_enrolled_students = 0)
+                }
+            })
+
+            // delete an offering block
+            await visitPage(page, '/courses')
+            await page.click('testEl=dashboard-settings-btn')
+            await page.click(`.offering-container[data-offering-id="${offeringIdWithEmptyStudents}"] >> testEl=delete-offering`)
+            await expect(page).toHaveSelector('testEl=delete-offering-modal')
+            await page.click('testEl=delete-offering-modal-btn')
+            await expect(page).not.toHaveSelector(`.offering-container[data-offering-id="${offeringIdWithEmptyStudents}"]`)
+
+            // go to a course
+            await page.click('testEl=dashboard-settings-btn', { force: true })
```

Comment:
why is force needed?  can you add a comment with the reason so we'll know if/when it's safe to remove it.

Reply:
it is not really needed, I was just testing out something and I forgot to take it out.